integration-tests/alfresco-community-52/overload.gradle

```diff
@@ -1,4 +1,4 @@
 ext {
     alfrescoBaseWar = 'org.alfresco:alfresco-platform:5.2.g@war'
-    alfrescoBaseImage = 'xeniteu/alfresco-repository-community:5.2.g'
+    alfrescoBaseImage = 'xenit/alfresco-repository-community:5.2.g'
```

Comment:
we should prefix public docker-hub images with `docker.io/`

Reply:
Done

Reply:
I meant `docker.io/xenit/....`

Reply:
Done